src/googmodule.ts

```diff
@@ -166,13 +166,11 @@ export function resolveIndexShorthand(
 function importPathToGoogNamespace(
     host: GoogModuleProcessorHost, file: ts.SourceFile, tsImport: string): ts.StringLiteral {
   let modName: string;
-  let isGoogImport = false;
```

Comment:
Unused?

Reply:
Yes, this was unused, and is now removed.